kratos/utilities/read_materials_utility.cpp

```diff
@@ -183,12 +184,15 @@ void ReadMaterialsUtility::AssignPropertyBlock(Parameters Data)
         if (KratosComponents<Variable<double> >::Has(variable_name)) {
             const Variable<double>& variable = KratosComponents<Variable<double>>().Get(variable_name);
             p_prop->SetValue(variable, value.GetDouble());
+            CheckIfOverwrittingValue(*p_prop, variable, value.GetDouble());
```

Comment:
Shouldn't this happen before you set the value?

Reply:
:D ups